.php_cs.dist

```diff
@@ -0,0 +1,75 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()
```

Comment:
pssst, you may disable autogenerated, 1MB+ files that are too slow to be processed